types/node.d.ts

```diff
@@ -7,6 +7,36 @@
 declare global {
   var isDevtools: boolean | undefined;
   var isLightrider: boolean | undefined;
+
+  // Augment Intl to include
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/getCanonicalLocales
+  namespace Intl {
+    var getCanonicalLocales: (locales?: string | Array<string>) => Array<string>;
+  }
+
+  interface Window {
```

Comment:
this is really weird in a file `node.d.ts` at first glance :). Maybe deserves a comment